tests/unit/components/sl-input-test.js

```diff
@@ -0,0 +1,210 @@
+import ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-input', 'Unit | Component | sl input', {
+    unit: true
+});
+
+
+test( 'Blur action is triggered when input loses focus', function( assert ) {
+    assert.expect( 1 );
+
+    const done = assert.async();
+
+    this.subject({
+        'blur': 'blur',
+        targetObject: {
+            blur() {
+                assert.ok(
+                    'blur was triggered'
+                );
+
+                done();
+            }
+        }
+    });
+
+    this.$( 'input' ).trigger( 'blur' );
+});
+
```

Comment:
Add a test for default values, specifically `clickToEdit`, `isTypeaheadSetup`, `suggestionNamePath` and `type`